### PR TITLE
Multistack example - vpc, s3 (ize-445)

### PR DESCRIPTION
**Pull request by kobrikx**
#### What's new:

 - Added multitask monorepo

### CODE REVIEW

**Review by psihachina — APPROVED** [review]

File: examples/multitask-monorepo/s3/s3.tf [hunk 1]
```diff
@@ -0,0 +1,28 @@
+module "s3_logs" {
```
Comment (AutomationD) [inline examples/multitask-monorepo/s3/s3.tf:1]: Let's remove all items that are not nessesary to show multistack example. Make sure to rebase too.

**Review by AutomationD — CHANGES_REQUESTED** [review]

File: examples/multitask-monorepo/vpc/main.tf [hunk 2]
```diff
@@ -0,0 +1,32 @@
+module "vpc" {
+  source  = "registry.terraform.io/terraform-aws-modules/vpc/aws"
+  version = "~> 3.0"
+
+  name = "${var.env}-vpc"
+  cidr = "10.0.0.0/16"
+
+  azs = [
+    "us-east-1a",
```
Comment (igorkotof) [inline examples/multitask-monorepo/vpc/main.tf:9]:
```suggestion
    "${var.aws_region}a",
```
And for other below

**Review by igorkotof — CHANGES_REQUESTED** [review]

File: examples/multitask-monorepo/vpc/main.tf (hunk 2 above)
Comment (kobrikx) [reply examples/multitask-monorepo/vpc/main.tf:9]: I can see this method for the first time. Could I use the old one?

File: examples/multitask-monorepo/main.tf [hunk 3]
```diff
@@ -0,0 +1,41 @@
+resource "aws_key_pair" "root" {
```
Comment (AutomationD) [inline examples/multitask-monorepo/main.tf:1]: Name of the file doesn't look right. Needs to be `examples/multistack-monorepo/main.tf`

File: examples/multitask-monorepo/vpc/main.tf (hunk 2 above)
Comment (AutomationD) [reply examples/multitask-monorepo/vpc/main.tf:9]: I'd say use a variable.

**Review by AutomationD — CHANGES_REQUESTED** [review]
Please remove all irrelevant pieces, and rebase.

**Review by AutomationD — CHANGES_REQUESTED** [review]
Looks good, please rebase to one commit then also let's have @igorkotof review.

**Review by igorkotof — APPROVED** [review]

**Review by AutomationD — APPROVED** [review]
LGTM. Thanks

**Pull request merged**